src/components/inference/custom-models-page.module.css

```
@@ -120,3 +120,9 @@
   padding: 48px 16px;
   text-align: center;
 }
+
+/* Body of the "model not supported" modal (see model-compatibility for the classification). */
+.rejectedModel {
+  font-weight: 600;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)